Examples/test-suite/cpp11_lambda_functions.i

```diff
@@ -27,7 +27,7 @@
 %warnfilter(SWIGWARN_CPP11_LAMBDA) Space1::Space2::lambda20;
 
 // throw is invalid in C++17 and later, only SWIG to use it
-#define TESTCASE_THROW0() throw()
+#define TESTCASE_THROW0() noexcept
```

Comment:
As the comment above notes, only SWIG gets fed this - it is testing SWIG's support for `throw()`, which we want to continue to test.